### ISSUE TITLE
Releases page - latest one is 1.0.0-alpha2

### ISSUE THREAD

**Issue by Misiu** [aspnet/SignalR#2696]
This issue is similar to https://github.com/aspnet/EntityFrameworkCore/issues/12550.
When navigating to https://github.com/aspnet/SignalR/releases latest release is 1.0.0-alpha2,

**Comment by anurse** [comment]
@danroth27 who manages this, is it still you? I notice other repos (https://github.com/aspnet/Mvc/releases) have the same problem.

**Comment by danroth27** [comment]
Yeah, we stopped generating releases on individual repos when we moved issue tracking for a bunch of repos into the Home repo. I guess technically we could still generate releases in repos that still maintain their own issue trackers.
